facia-press/app/frontpress/FapiFrontPress.scala

```diff
@@ -254,12 +254,10 @@ trait FapiFrontPress extends Logging with ExecutionContexts {
         commercial = {
           val tag = itemResp flatMap (_.tag)
           val section = itemResp flatMap (_.section)
-          Some(CommercialProperties(
-            editionBrandings =
-              tag.map(EditionBranding.fromTag) orElse section.map(EditionBranding.fromSection) getOrElse Nil,
-            editionAdTargetings =
-              tag.map(EditionAdTargeting.fromTag) orElse section.map(EditionAdTargeting.fromSection) getOrElse Nil
-          ))
+          tag.map(CommercialProperties.fromTag) orElse
+            section.map(CommercialProperties.fromSection) orElse
+            CommercialProperties.forNetworkFront(path) orElse
```

Comment:
should this be 3rd priority? seems like it should be higher priority than `fromTag` and `fromSection`. Maybe comment to explain why?